master/buildbot/test/fake/fakedb.py

```diff
@@ -90,6 +91,10 @@ def __init__(self, **kwargs):
         for k, v in iteritems(self.values):
```

Comment:
well, as this is "only" test code, I dont think we should bother be annoying on that, I prefer we do like this. did you try to remove that code? I would be you have a few thousands lines to update..